values-schema.yaml

```diff
@@ -207,6 +214,61 @@ definitions:
       tag:
         type: string
     title: Container image
+  jobs:
+    description: You may pass an array of Job objects. The Kubernetes API accepts
+      a Job specification, which is meant for run-to-completion apps (e.g. batch
+      Jobs).
+    additionalProperties: false
+    items:
+      additionalProperties: false
+      properties:
+        deployAlways:
+          description: |
+            May give a boolean value. This is false by default.
+            If true, this job will run every `helmfile apply`.
```

Comment:
There are many helmfile commands that would ultimately trigger deployment. I suggest to rephrase to:
`If true, this job will run every deployment `

Reply:
I don't see any response or code change. I do not understand why it is marked as resolved